docs/content/guides/upgrade-and-migration/migrating-from-12.4-to-13.0.md

```diff
@@ -0,0 +1,123 @@
+---
+id: fgjqkigc
+title: Migrating from 12.4 to 13.0
+metaTitle: Migrate from 12.4 to 13.0 - JavaScript Data Grid | Handsontable
+description: Migrate from Handsontable 12.4 to Handsontable 13.0, released on June 22, 2023.
+permalink: /migration-from-12.4-to-13.0
+canonicalUrl: /migration-from-12.4-to-13.0
+pageClass: migration-guide
+react:
+  id: tvum3ibg
+  metaTitle: Migrate from 12.4 to 13.0 - React Data Grid | Handsontable
+searchCategory: Guides
+---
+
+# Migrate from 12.4 to 13.0
+
+Migrate from Handsontable 12.4 to Handsontable 13.0, released on June 22, 2023.
+
+[[toc]]
+
+::: only-for javascript
+
+## Update to Angular 12 or higher
+
+Handsontable 13.0 requires Angular 12 or higher. If you're using an older version of Angular, you need to update it.
```

Comment:
@kirszenbaum This is probably worth rephrasing as well.

Reply:
Ok, done